src/Symfony/Component/Notifier/Bridge/MessageBird/MessageBirdTransport.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MessageBird;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class MessageBirdTransport extends AbstractTransport
+{
+    protected const HOST = 'rest.messagebird.com';
+
+    private $authToken;
+    private $from;
+
+    public function __construct(string $authToken, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
```suggestion
    public function __construct(string $token, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

because you use `TOKEN` in the README, lets use `$token` across the whole bridge, thanks